assets/js/main.js

```diff
@@ -0,0 +1,323 @@
+/////////////////////////////
+// Declares DOM elements
+/////////////////////////////
+
+var listDrawPile = document.querySelector('.DrawPile')

```

Comment:
same like variables, classname attribute should't start with capital (e.g. '.drawPile' not '.DrawPile')